test/minikube/minikube_long_restore_test.go

```diff
@@ -351,38 +351,64 @@ func TestKubernetesRestoreWithStorageGroups(t *testing.T) {
 			"database.sm.resources.requests.memory": testlib.MINIMAL_VIABLE_ENGINE_MEMORY,
 			"database.te.resources.requests.cpu":    "250m",
 			"database.te.resources.requests.memory": testlib.MINIMAL_VIABLE_ENGINE_MEMORY,
-			"database.sm.hotCopy.replicas":          "2",
 			"database.te.logPersistence.enabled":    "true",
+			"database.sm.storageGroup.enabled":      "true",
 			"database.env[0].name":                  "NUODB_DEBUG",
 			"database.env[0].value":                 "debug",
 			// include both HCSMs as each of them is serving separate storage group
-			"database.sm.hotCopy.backupGroups.group0.labels": "role hotcopy",
+			"database.sm.hotCopy.backupGroups.all-sg.labels": "role hotcopy",
 		},
 	}
 
 	defer testlib.Teardown(testlib.TEARDOWN_DATABASE)
-	databaseChartName := testlib.StartDatabase(t, namespaceName, admin0, &databaseOptions)
 
 	// Generate diagnose in case this test fails
 	testlib.AddDiagnosticTeardown(testlib.TEARDOWN_DATABASE, t, func() {
 		testlib.GetDiagnoseOnTestFailure(t, namespaceName, admin0)
 	})
 
+	// install database primary release which serves sg0
+	databaseOptions.SetValues["database.sm.storageGroup.name"] = "sg0"
+	sg0ReleaseName := testlib.StartDatabase(t, namespaceName, admin0, &databaseOptions)
+
+	// install database secondary release which serves sg1
+	databaseOptions.SetValues["database.primaryRelease"] = "false"
+	databaseOptions.SetValues["database.te.enablePod"] = "false"
+	databaseOptions.SetValues["database.sm.storageGroup.name"] = "sg1"
+	sg1ReleaseName := testlib.StartDatabase(t, namespaceName, admin0, &databaseOptions)
+
+	// set the total number of engines as in the test utilities they are
+	// inferred from the values
+	databaseOptions.SetValues["database.te.enablePod"] = "true"
+	databaseOptions.SetValues["database.te.replicas"] = "1"
+	databaseOptions.SetValues["database.sm.hotCopy.replicas"] = "2"
```

Comment:
What is this for? It looks like we're updating the Helm values after we have done the deployment. Is this to prevent checks from breaking that inspect the Helm values?